crates/threshold-bls-ffi/src/lib.rs

```diff
@@ -2,3 +2,21 @@ pub mod ffi;
 
 #[cfg(feature = "wasm")]
 pub mod wasm;
+
+use threshold_bls::{
+    curve::zexe::PairingCurve as Bls12_377,
+    sig::{bls::G2Scheme, Scheme},
+    Index,
+};
+
+pub(crate) type SigScheme = G2Scheme<Bls12_377>;
+pub(crate) type PublicKey = <SigScheme as Scheme>::Public;
+pub(crate) type PrivateKey = <SigScheme as Scheme>::Private;
+pub(crate) type Signature = <SigScheme as Scheme>::Signature;
+
+pub(crate) const VEC_LENGTH: usize = 8;
+pub(crate) const SIGNATURE_LEN: usize = 48;
+pub(crate) const PUBKEY_LEN: usize = 96;
+pub(crate) const PRIVKEY_LEN: usize = 32;
+pub(crate) const PARTIAL_SIG_LENGTH: usize =
+    VEC_LENGTH + SIGNATURE_LEN + std::mem::size_of::<Index>();
```

Comment:
@kobigurk These extra 8 bytes can be removed by moving the `marshal_len` method, which was part of the `Encodable` trait, to the `Element` trait. Then, I'd have to implement a serde serializer for `struct PartialSig<E> { index: Index, sig: E}` where `E: Element`, and just tell it to deserialize it as before. 

(This logic would be the same as the extract/inject index we had before, but in serde syntax)